src/transformers/models/bridgetower/image_processing_bridgetower.py

```diff
@@ -125,17 +126,17 @@ class BridgeTowerImageProcessor(BaseImageProcessor):
     Constructs a BridgeTower image processor.
 
     Args:
-        do_resize (`bool`, *optional*, defaults to `True`):
+        do_resize (`bool`, defaults to `True`, *optional*, defaults to `True`):
```

Comment:
I think something funny happened with the formatting here 

```suggestion
        do_resize (`bool`, *optional*, defaults to `True`):
```

Reply:
hmm, looks like a make fixup mishap yes